index.d.ts

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface SourceSpec {
```

Comment:
Is there any point `export` ing `SourceSpec` and `TrackSpec`? They are only used in this file.

Reply:
> They are only used in this file.
Sure?
If you want to prepare an object of type `SourceSpec` somewhere in your source you'll be happy to be able to import that type.